Update Compaction Strategy example

jmark99 opened the pull request:
Update the Compaction Strategy example to make use of the new Compaction Configuration API.

Closes #82

jmark99 committed:
Update Compaction Strategy example

Update the Compaction Strategy example to make use of the new Compaction
Configuration API.

Manno15 reviewed (approved):
Changes look good. Confirmed the example works and the amount of detail is vastly improved. I assume this example will also be updated to the main branch as well?

dlmarion commented on docs/compactionStrategy.md:
````diff
@@ -16,64 +16,137 @@ limitations under the License.
- $ accumulo shell -u <username> -p <password> -e "config -t examples.test1 -s table.majc.compaction.strategy.opts.large.compress.threshold=100M"
- $ accumulo shell -u <username> -p <password> -e "config -t examples.test1 -s table.majc.compaction.strategy.opts.large.compress.type=gz"
-```
+ - Limit the compaction throughput to 40M.
````
Below you have this as `40MB/s`. Using that here would make this more clear.

jmark99 replied:
@dlmarion , with this clarification updated, are you good with me merging this example?

dlmarion replied:
Yes I believe so. Looks good. Thanks!

dlmarion reviewed (commented):
Looks good, one suggested change to make things clear about the rate limit.

jmark99 committed:
Update documentation to clarify rate limit value

Update documention to add consistency in the way the rate limit is
references.

Modified '40M' to '40MB/s' so  both references to the limit use the same
format.
